docs/RELEASE.md

```diff
@@ -3,24 +3,24 @@
 To release a new version of Kuberhealthy:
 
 1. Ensure that all issues in the release milestone are completed
-1. Verify that the master branch has all the latest changes that have been tested and verified
-1. After any running `Build and Push kuberhealthy Unstable` action is completed, tag a release candidate from the build done in master as unstable
+2. Verify that the master branch has all the latest changes that have been tested and verified
```

Comment:
This is actually supposed to be 1. on every line.  This causes markdown to automatically number each thing.